regression/scripts/AVL/test_AVL.py

```diff
@@ -49,7 +49,7 @@ def main():
     configs_analyses = analyses_setup(configs)
 
     
-    run_new_regression = False
+    run_new_regression = True
```

Comment:
I think this is the line that is messing up the regresssion

Reply:
Ah yes, thanks